samples/nrf9160/modem_shell/src/gnss/gnss_interface.c

```diff
@@ -456,28 +451,176 @@ static int gnss_enable_all_nmeas(void)
 		NRF_MODEM_GNSS_NMEA_RMC_MASK);
 }
 
-#if defined(CONFIG_SUPL_CLIENT_LIB)
+#if defined(CONFIG_NRF_CLOUD_AGPS)
+static int serving_cell_info_get(struct lte_lc_cell *serving_cell)
```

Comment:
We would have `static int link_api_xmonitor_read` which could be made available. Or that's what was done in Location library in similar case.
Maybe here it's simpler to use modem_info and not create dependency to link module...

Reply:
I propose that we keep it like this for now.